client/components/loading-ellipsis/index.tsx

```diff
@@ -1,7 +1,7 @@
 import classnames from 'classnames';
-import type { ReactNode } from 'react';
+import type { ReactElement } from 'react';
 
-export function LoadingEllipsis( { className }: { className?: string } ): ReactNode {
+export function LoadingEllipsis( { className }: { className?: string } ): ReactElement {
```

Comment:
Why the type change? Was there an error?

Reply:
this change was introduced by the branched off branch, but I did a quick search and it seems that we use `:ReactElement` when creating a React component.

Reply:
Got it, that makes sense!